src/main/java/co/inventorsoft/academy/dao/MatchDAO.java

```diff
@@ -0,0 +1,117 @@
+     *
+     * @param match input Match
+     */
+    @Override
+    public int addMatch(Match match) {
+        int result = -1;
+        try (Connection connection = MyJDBC.getConnection();
+             PreparedStatement preparedStatement = connection.prepareStatement(INSERT_MATCH)) {
+            preparedStatement.setString(1, match.getRound());
+            preparedStatement.setInt(2, match.getTeam1());
+            preparedStatement.setInt(3, match.getTeam2());
+            preparedStatement.setInt(4, match.getPointsTeam1());
+            preparedStatement.setInt(5, match.getPointsTeam2());
+            ResultSet rs = preparedStatement.executeQuery();
+            while(rs.next()){
+                result = rs.getInt("id");
+            }
+        } catch (SQLException e) {
+            MyJDBC.printSQLException(e);
+        }
+        return result;
+    }
+
+    /**
+     * This method get Match from database by input index
+     *
+     * @param id input index
+     * @return Match
+     */
+    @Override
+    public Match getMatch(Integer id) {
+       Match match = null;
+        try (Connection connection = MyJDBC.getConnection();
+             PreparedStatement preparedStatement = connection.prepareStatement(SELECT_MATCH_BY_ID)) {
+            preparedStatement.setInt(1, id);
+            ResultSet rs = preparedStatement.executeQuery();
+            while (rs.next()) {
+                String round = rs.getString("round");
+                int team1 = rs.getInt("team1");
+                int team2 = rs.getInt("team2");
+                int score1 = rs.getInt("points_team1");
+                int score2 = rs.getInt("points_team2");
+                 match = new Match(round, team1, team2, score1, score2);
+            }
+        } catch (SQLException e) {
+            MyJDBC.printSQLException(e);
+        }
+        return match;
+    }
+
+    /**
+     * This method create list of matches from database
+     */
+    @Override
+    public List<Match> getMatches() {
+        List<Match> matches = new ArrayList<>();
+        try (Connection connection = MyJDBC.getConnection();
+             PreparedStatement preparedStatement = connection.prepareStatement(SELECT_MATCH)) {
+            ResultSet rs = preparedStatement.executeQuery();
+            while (rs.next()) {
+                String round = rs.getString("team");
+                int team1 = rs.getInt("team1");
+                int team2 = rs.getInt("team2");
+                int score1 = rs.getInt("points_team1");
+                int score2 = rs.getInt("points_team2");
+                Match match = new Match(round, team1, team2, score1, score2);
+                matches.add(match);
+            }
+        } catch (SQLException e) {
+            MyJDBC.printSQLException(e);
+        }
+        return matches;
+    }
+
+    /**
+     * This method delete Match from database
+     *
+     * @param id input Match id
+     */
+    @Override
+    public void deleteMatch(Integer id) {
+
```

Comment:
If you think that method is not needed but you should implement it, it is better to inform user that this method is not implemented or not allowed to use.
`java.lang.UnsupportedOperationException`